src/heatmap.ts

```diff
@@ -16,7 +16,7 @@ import {IVector} from 'phovea_core/src/vector';
 import {toSelectOperation, defaultSelectionType} from 'phovea_core/src/idtype';
 
 
-function defaultColor(value: any) {
+export function defaultColor(value: any) {
```

Comment:
Or it might be even better to create an interface for the value property that we can use for typing the functions (e.g., `defaultColor(value: IDataDescriptionValue)`).